packages/dds/tree/src/test/utils.ts

```diff
@@ -227,9 +229,19 @@ export class TestTreeProvider {
 	 * _i_ is the index of the tree in order of creation.
 	 */
 	public async createTree(): Promise<ISharedTree> {
+		const configProvider = (settings: Record<string, ConfigTypes>): IConfigProviderBase => ({
+			getRawConfig: (name: string): ConfigTypes => settings[name],
+		});
+		const testContainerConfig: ITestContainerConfig = {
+			loaderProps: {
+				configProvider: configProvider({
+					"Fluid.Container.enableOfflineLoad": true,
```

Comment:
Regarding your comment in the description: I think it's likely fine to have this set to `true` in all cases; it is a feature flag and we want to test the feature.